src/App.js

```diff
@@ -1,16 +1,42 @@
 import React from 'react';
 import './App.scss';
+import { Counter } from './components/Counter';
 
 class App extends React.Component {
-  addOne = () => {};
+  state = {
+    count: 0,
+  };
 
-  add100 = () => {};
+  addOne = () => {
+    this.setState(state => ({
```

Comment:
```suggestion
    this.setState(prevState => ({
```
або `previousState`. Тому що ти змінюєш попередній `state`, так зрозуміліше який стан мається на увазі